commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/CommandAPIBukkit.java

```diff
@@ -277,14 +277,12 @@ private String[] getUsageList(RegisteredCommand currentCommand) {
 
 	void updateHelpForCommands(List<RegisteredCommand> commands) {
 		Map<String, HelpTopic> helpTopicsToAdd = new HashMap<>();
-		Set<String> namespacedCommandNames = new HashSet<>();
 
 		for (RegisteredCommand command : commands) {
+			if (getPaper().isPaperBrigAPI() && !command.shouldGenerateHelpTopic()) continue;
```

Comment:
Could call `getPaper().isPaperBrigAPI()` once, although that may be slightly less readable for not a big deal:
```suggestion
		boolean isPaperBrigAPI = getPaper().isPaperBrigAPI();
		for (RegisteredCommand command : commands) {
			if (isPaperBrigAPI && !command.shouldGenerateHelpTopic()) continue;
```